app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java

```diff
@@ -394,17 +394,18 @@ public static Bitmap getBitmapFromDrawable(Context context, @DrawableRes int dra
     }
 
     /**
-     * Return a round float value with 2 decimals
+     * Return a round float value <b>with 2 decimals</b>
+     * <b>BE CAREFUL:</b> THE METHOD DOESN'T CHECK THE NUMBER AS A NUMBER.
      *
      * @param value float value
-     * @return round value or 0 if the value is empty or equals to 0
+     * @return round value <b>with 2 decimals</b> or 0 if the value is empty or equals to 0
      */
     public static String getRoundNumber(String value) {
         if ("0".equals(value)) {
             return value;
         }
 
-        if (isEmpty(value)) {
+        if (value == null || value.length() == 0) {
```

Comment:
Why not use `TextUtils.isEmpty()` ? (native from android)

Reply:
Because I can't seem to figure out why when i pass the string `""` it doesn't return true.